src/Common/src/Interop/Windows/advapi32/Interop.ClearEventLog.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Advapi32
+    {
+        [DllImport(Libraries.Advapi32, CharSet = CharSet.Unicode, SetLastError = true)]
+        public static extern bool ClearEventLog(SafeHandle hEventLog, HandleRef lpctstrBackupFileName);
```

Comment:
It does not really make sense to use `HandleRef` for a string. Should this be changed to just `string` ?

Reply:
Here is the using: `bool success = Interop.Advapi32.ClearEventLog(readHandle, NativeMethods.NullHandleRef);`

Reply:
This can be just `Interop.Advapi32.ClearEventLog(readHandle, null)` once you change the `HandleRef` to `string` in the `ClearEventLog` signature.